.github/workflows/test-pull-requests.yml

```diff
@@ -1,30 +1,21 @@
-name: Test pull requests and generate preview link
```

Comment:
Hey Tharindu37! Thank you for submitting your PR. I noticed that the test build has been removed from the workflow. Also, the preview link should be https://pr-${{ github.event.number }}-sef-handbook.surge.sh. Could you make the update by only changing the token and keeping the rest as it is? Thanks!